.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,33 @@
+<!--
+Thank you for your Pull Request! Please describe the problem this PR fixes and a summary of the changes made.
+Link to any relevant issues, code snippets, or other PRs.
+
+For trivial changes, this template can be ignored in favor of a short description of the changes.
+-->
+
+## Problem
+
+<!--
+Describe the issue this PR is solving
+-->
+
+**Issue number, if available:**
+
+## Changes
+
+<!--
+Summarize the changes introduced in this PR. This is a good place to call out critical or potentially problematic parts of the change.
+-->
+
+## Checklist
```

Comment:
```suggestion
## Changes

<!--
Summarize the changes introduced in this PR. This is a good place to call out critical or potentially problematic parts of the change.
-->

## Validation
<!--
Describe how changes in this PR are validated. This may include, tests, e2e tests, manual verifications, test builds etc.
--->

## Checklist
```

Reply:
I didn't use your exact wording, but I added a Validation section with an explanation comment. LMK if it looks good.

Reply:
Looks good. I left one more suggestion below.